src/ucp/wireup/select.c

```diff
@@ -1218,6 +1229,12 @@ static double ucp_tl_iface_bandwidth_ratio(ucp_context_h context,
     return ucs_max(1e-5, 1.0 - ratio);
 }
 
+static double ucp_wireup_fp8_pack_unpack_bandwidth(double bandwidth)
```

Comment:
pls reuse code with ucp_wireup_fp8_pack_unpack_bw and put this function after ucp_wireup_fp8_pack_unpack_latency